src/game_io/game_cmd_queue_packet.cc

```diff
@@ -95,23 +95,22 @@ void GameCmdQueuePacket::write(FileSystem& fs, Game& game, MapObjectSaver* const
 
 	while (nhandled < cmdq.ncmds_) {
 		// Make a copy, so we can pop stuff
-		std::priority_queue<CmdQueue::CmdItem> p = cmdq.cmds_[time % kCommandQueueBucketSize];
+		std::multiset<CmdQueue::CmdItem> p = cmdq.cmds_[time % kCommandQueueBucketSize];
 
 		while (!p.empty()) {
-			const CmdQueue::CmdItem& it = p.top();
-			if (it.cmd->duetime() > time) {
+			if (p.begin()->cmd->duetime() > time) {
```

Comment:
please extract p.bgein as varibale, I have no idea what this is.
e.g. CmdItem nextCommand = p.begin();